### PR TITLE
Feature/395

### PR DESCRIPTION
**Pull request by tatikozh**

### CODE REVIEW

**Review by mdconatser — DISMISSED** [review]

File: lib/ui/mail_view.dart [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+    ),
+  );
+}
+
+class MailPiece {
+  final DateTime _timeStamp;
+  final String _sender;
+  final String _imageText;
+  final Image _mailImage;
+  final String _mailDescription;
```
Comment (umgcjack) [inline lib/ui/mail_view.dart]: You can make these public, which allows you to get rid of the getters below!
`final` variables are not settable, so making a `final` variable public exposes the getter but not a setter.

**Review by umgcjack — COMMENTED** [review]
Awesome work!

File: lib/ui/mail_view.dart (hunk 1 above)
Comment (tatikozh) [reply lib/ui/mail_view.dart]: @umgcjack thankyou! I'll edit it. However, those variables are within the MailPiece class that the actual MailPice model will replace

File: lib/ui/mail_view.dart [hunk 2]
```diff
@@ -0,0 +1,111 @@
...
+  );
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
+      bottomNavigationBar: const BottomBar(),
+      appBar: AppBar(
+        title: Text('Search Results',  style:
+        TextStyle(fontWeight: _commonFontWeight, fontSize: _commonFontSize),
+        ),
+        backgroundColor: _buttonColor,
```
Comment (shaneknows) [inline lib/ui/mail_view.dart:32]: Did you mean to have this in the TextStyle initiation above? it doesn't look like AppBar has a backgroundColor property.

File: lib/ui/search.dart [hunk 3]
```diff
@@ -0,0 +1,223 @@
...
+                                      fontWeight: _commonFontWeight,
+                                      color: Color.fromRGBO(51, 51, 102, 100)
+                                  ),
+                                ),
+                                SizedBox(
+                                  height: 50.0,
+                                  child:
+                                  ElevatedButton.icon(
+                                    style: ElevatedButton.styleFrom(
+                                      backgroundColor: Color.fromRGBO(51, 51, 102, 1.0),
```
Comment (shaneknows) [inline lib/ui/search.dart:99]: I'm getting an error with this backgroundColor saying there is no named parameter

**Review by shaneknows — COMMENTED** [review]
Good work! A few backgroundColor properties were erroring out for me, but after commenting them out  it looks close to what I was expecting. I'm not sure if I was missing something to get this working or if it truly is missing.

File: lib/ui/search.dart (hunk 3 above)
Comment (tatikozh) [reply lib/ui/search.dart:99]: @shaneknows I was just looking into that and it works well on my side with no errors

**Comment by mdconatser** [comment]
I was having trouble with the backgroundColor as well, I had to change it to primary to get it to work. Not sure how we are having different library versions or something...it seems to build fine on the server

**Review by mdconatser — APPROVED** [review]

File: lib/ui/search.dart (hunk 3 above)
Comment (shaneknows) [reply lib/ui/search.dart:99]: Weird I see it working with the build artifact, but I cannot get this to work locally without changing this to primary

**Review by shaneknows — APPROVED** [review]

**Pull request merged**